openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestDescriptionPage/RequestDescriptionPage.tsx

```diff
@@ -116,7 +122,7 @@ const RequestDescription = () => {
       const data: CreateThread = {
         from: currentUser?.name as string,
         message: value.title || taskMessage,
-        about: getEntityFeedLink(entityType, decodedEntityFQN, getTaskAbout()),
+        about: getEntityFeedLink(entityType, entityFQN, getTaskAbout()),
```

Comment:
What's rsn for changing decodedFQN with FQN

Reply:
It's the decoded fqn itself but in the case of `entityType: Table` it's being modified with `getTaskEntityFQN` and the return value is saved in `entityFQN`